client_code/forms/routes/ballot/Index/__init__.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, **properties):
         self.init_tabulator()
 
     def show_ballot(self, uuid):
+        session.sync_ballot(session.SCROLL_STORE, session.LOCAL_STORE, uuid)
```

Comment:
If `session.sync_ballot` returned the Ballot instance, the following line would not be necessary here.